7-loops/For Each

```diff
@@ -0,0 +1,49 @@
+New Feature added in ES 6 
+
+
+The forEach() method in Javascript calls back  a function for each element in an array .
+
+
+The forEach() method is not executed for empty elements.
+
+
+Example :
+
+let arr = ['Goutham' , 'Korla' , 'Prabhath' , 'Bogam'];
+
+ forEach is a function which accepts a function which is required
+
+and a current item which is also required ,
+
+index which is optional,
+
+array which is optional ,
+
+
+This parameter is optional, it holds the context to be passed as this to be used while executing the callback function. If the context is passed, it will be used like this for each invocation of the callback function, otherwise undefined is used as default.
+
+1.  syntax :
+
+arr.forEach(callback(element, index, arr), thisValue)
+
+
+
+2 . arr.forEach( function ( item ) {
+
+console.log(item);  //Goutham Korla Prabhath Bogam will be printed
+
+} )
+ 
+ 
+ 3 .  let us consider other examples as well
+ 
```

Comment:
there is no challenge here, you must read the [CONTRIBUTING.md](https://github.com/inspirezonetech/TeachMeJavaScriptLikeIm5/blob/main/CONTRIBUTING.md) guide please.

Reply:
Now I have made some changes